pkg/agent/datamodel/types.go

```diff
@@ -847,6 +847,14 @@ func (p *Properties) IsVHDDistroForAllNodes() bool {
 	return true
 }
 
+func (config *NodeBootstrappingConfiguration) Is2204VHD() bool {
+	if config.AgentPoolProfile.Distro == AKSUbuntuContainerd2204 || config.AgentPoolProfile.Distro == AKSUbuntuContainerd2204Gen2 {
+		return true
+	}
+
+	return false
```

Comment:
```suggestion
        distro := config.AgentPoolProfile.Distro
        return distro == AKSUbuntuContainerd2204  || distro == AKSUbuntuContainerd2204Gen2 
```

could be nice to add a helper for e.g. IsCgroupv2Distro maybe

Reply:
(nit)

Reply:
addressed and added a helper function Is2204VHDDistro